README.md

```diff
@@ -1,2 +1,170 @@
 # orchestrator-helm-chart
-Helm chart to deploy the Orchestrator solution suite, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator, Knative Eventing, Knative Serving, Data Index and Job Service.
+
+This chart will deploy the following on the target OpenShift cluster:
+  - Janus IDP backstage
+  - SonataFlow Operator
+  - OpenShift Serverless Operator
+  - Knative Eventing
+  - Knative Serving
+  - Data Index Service
+  - Job Service
+  - PostgreSQL Databases 
+  - Sample workflow (event-timeout)
+
+## Installation
+Build helm dependency:
+```console
+$ git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
+$ cd orchestrator-helm-chart/charts
+$ helm dep update orchestrator
+```
+Perform a first pass install:
+```console
+$ helm install orchestrator orchestrator
+NAME: orchestrator
+LAST DEPLOYED: Thu Nov  2 09:02:29 2023
+NAMESPACE: jianrong
+STATUS: deployed
+REVISION: 1
+```
+Wait for about 1 minute, and then check to make sure the SonataFlow operator and OpenShift Serverless operator have started.
+```console
+$ oc wait --for=condition=Available deploy sonataflow-operator-controller-manager -n openshift-operators
+deployment.apps/sonataflow-operator-controller-manager condition met
+$ oc wait --for=condition=Available deploy knative-openshift -n openshift-serverless
+deployment.apps/knative-openshift condition met
+```
```

Comment:
At this point of the instruction, there are 2 deployments that failed to start (postgres and backstage).
Seems the reason is a failure to create a PVC.
Should there be a default storage class as a requirement for this solution to work?
We should have all of the requirements documented.

Reply:
At this point of the installation, the backstage has errors:
```
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-11-06T13:47:46.898Z orchestrator error Error when checking SonataFlow health: Error: Unable to execute query. type=plugin
2023-11-06T13:47:46.899Z orchestrator info Auto starting SonataFlow through: docker run --add-host host.docker.internal:host-gateway --rm -p 8899:8080 -v /tmp/orchestrator/repository/workflows:/home/kogito/serverless-workflow-project/src/main/resources -e KOGITO.CODEGEN.PROCESS.FAILONERROR=false -e QUARKUS_EMBEDDED_POSTGRESQL_DATA_DIR=/home/kogito/persistence quay.io/kiegroup/kogito-swf-devmode-nightly:main-2023-08-30 type=plugin
2023-11-06T13:47:46.935Z orchestrator info Checking SonataFlow health at: http://localhost:8899/q/health type=plugin
2023-11-06T13:47:46.940Z orchestrator error SonataFlow process error: Error: spawn docker ENOENT type=plugin
2023-11-06T13:47:46.942Z orchestrator info SonataFlow process exited with code -2 type=plugin
2023-11-06T13:49:02.000Z orchestrator error Error when checking SonataFlow health: Error: Unable to execute query. type=plugin
2023-11-06T13:49:02.001Z orchestrator error SonataFlow is not up. Check your configuration. type=plugin
Using backend plugin orchestrator...
Adding router api to backend...
2023-11-06T13:49:02.017Z app info Serving static app content from /opt/app-root/src/packages/app/dist type=plugin
2023-11-06T13:49:02.639Z app info Injecting env config into module-backstage.0b174eef.js type=plugin
2023-11-06T13:49:03.466Z app info Storing 0 updated assets and 453 new assets type=plugin
Adding router app to backend...
Adding router metrics to backend...
2023-11-06T13:49:05.353Z backstage info Listening on :7007
2023-11-06T13:49:45.661Z catalog warn file /opt/catalog-entities/all.yaml does not exist type=plugin entity=location:default/generated-28cca00eef20648da516bda20fe0e71b7b7364ba location=file:/opt/catalog-entities/all.yaml
```

Reply:
>  Should there be a default storage class as a requirement for this solution to work?

I confirm that it's a requirement. 
+1 to document it

Reply:
> At this point of the instruction, there are 2 deployments that failed to start (postgres and backstage). Seems the reason is a failure to create a PVC. Should there be a default storage class as a requirement for this solution to work? We should have all of the requirements documented.

Yes, I added it to the "prerequisites" section.